cli/README.md

```diff
@@ -26,7 +26,7 @@ To use the Xata CLI, you'll have to **authenticate** with it to access your work
 
 To authenticate globally, across your entire system, run `xata auth login`. This will give you two options:
 
-- **Create a new API Key**. This will open your browser and, when you're logged in to Xata, allow you to create a new [API key](https://docs.xata.io/concepts/api-keys) for use with the CLI.
+- **Create a new API Key**. This will open your browser and, when you're logged in to Xata, allow you to create a new [API key](https://xata.io/docs/concepts/api-keys) for use with the CLI.
```

Comment:
```suggestion
- **Create a new API Key**. This will open your browser and, when you're logged in to Xata, allow you to create a new [API key](https://xata.io/docs/getting-started/api-keys) for use with the CLI.
```